docs/en/sql-reference/functions/ip-address-functions.md

```diff
@@ -244,10 +244,26 @@ SELECT IPv6CIDRToRange(toIPv6('2001:0db8:0000:85a3:0000:0000:ac1f:8001'), 32);
 └────────────────────────────────────────────────────────────────────────┘
 ```
 
-## toIPv4(string)
+## toIPv4
 
 An alias to `IPv4StringToNum()` that takes a string form of IPv4 address and returns value of [IPv4](../data-types/ipv4.md) type, which is binary equal to value returned by `IPv4StringToNum()`.
 
+**Syntax**
```

Comment:
In l. 249, it would be nice to replace "`IPv4StringToNum`" by a link.

Reply:
In l. 249, it is weird to say that `toIPv4` is an "alias" of `IPv4StringToNum` when the very example in l. 268 demonstrates it is not an alias (both return different types).